datasource/etcd/cache/etcd/indexer_cache.go

```diff
@@ -18,23 +18,23 @@ package etcd
 
 import (
 	"fmt"
+	"github.com/apache/servicecomb-service-center/datasource/etcd/cache"
 
+	registry "github.com/apache/servicecomb-service-center/datasource/etcd/client"
```

Comment:
since you rename registry to client, you use alias to replace client again, wht is wierd. plz use registry